src/app/logging.rs

```diff
@@ -58,6 +62,8 @@ pub struct LogMessage {
 pub struct Logger {
     log_file: Option<File>,
     log_filepath: Option<String>,
+    latest_log_file: Option<File>,
+    latest_log_filepath: Option<String>,
```

Comment:
At first, I thought this field was unnecessary, as it was only used when creating the latest log file in `init_log_file()`. This prompted me to think that the field `log_filepath` was also useless, but I realized that @OJarrisonn did this as this field is used in `flush()`.

TLDR: shouldn't we also flush the latest log like we do with a normal log? Files aren't buffered in Rust and are written straightly (otherwise, we couldn't watch the file), but to maintain the pattern and be as safe as possible, maybe we need to enforce flushing here to.

Reply:
Forget, I was mistaken about what I wrote.